TTS_xwing/src/BehaviourDB.ttslua

```diff
@@ -145,7 +145,7 @@ BehaviourDB.rule_sets[1].ships = {}
 -- TIE/ln Fighter
 ---- This logic is shared between Imperial and Rebel. See Z-95 for an example on how to split them if necessary.
 BehaviourDB.rule_sets[1].ships['tielnfighter'] = {}
-BehaviourDB.rule_sets[1].ships['tielnfighter'].target_selection = {[1]='ClosestInArc', [2]='Closest'}
+BehaviourDB.rule_sets[1].ships['tielnfighter'].target_selection = {[1]='TargetForStrikeAI', [2]='ClosestInArc', [3]='Closest'}
```

Comment:
That sounds good if it's not too much of a hack. I didn't do all the ships for the very reason that I figured someone who knows what they're doing might have a better suggestion.
I did do the TIE/Advanced x1 since I wanted a ship with Target Lock. By Flight Group Alpha (FGA) rules the StrikeAI ship won't Target Lock a non-Strike target, but will take shots of opportunity on a non-Strike Target at Range-1.
I felt that letting the Strike AI continue to pursue the target it has a Lock on was best. So the TargetForStrikeAI() method for a ship with Target Lock comes second after finding there isn't a Locked Target.

Reply:
We would have to modify this code to make it implicit. I don't see how to do it without asking for target locks though.

        -- Find this ship's target.
        local target_ship = nil
        for _, target_selection_function in ipairs(ship_behaviour.target_selection) do
            target_ship = AIModule.TargetSelectionFunctions[target_selection_function](ship)
            if target_ship ~= nil then
                break
            end
        end
        AIModule.current_move.target = target_ship